R/lineage.R

```diff
@@ -358,21 +358,21 @@ ipg2lin <- function(accessions, ipg_file,
     if (length(refseq_rows) != 0) {
         refseq_ipg_dt <- ipg_dt[refseq_rows, ]
         refseq_lins <- GCA2lin(refseq_ipg_dt,
-            assembly_path = refseq_assembly_path,
+            .data$assembly_path = refseq_assembly_path,
```

Comment:
The `.data` prefix is not required in this instance, as `assembly_path` is an argument to `GCA2lin()`.